### PR TITLE
chore: merged picklist item

### PR DESCRIPTION
**Pull request by mrunmay-epi**
Task link: https://bloomstack.com/desk#Form/Task/TASK-2021-00250
Bloomstack core PR link:  https://github.com/Bloomstack/bloomstack_core/pull/635

Added following fields:
Pick List Item-cb_reference

SO Customer

### CODE REVIEW

**Comment by sahil28297** [comment]
@mrunmay-epi Where's the before after screenshots??

File: erpnext/stock/doctype/pick_list_item/pick_list_item.json [hunk 1]
```diff
@@ -200,10 +202,21 @@
    "label": "COA Batch",
    "options": "Batch",
    "read_only": 1
+  },
+  {
+   "fieldname": "cb_reference",
+   "fieldtype": "Column Break",
+   "label": "Pick List Item-cb_reference"
```
Comment (sahil28297) [inline erpnext/stock/doctype/pick_list_item/pick_list_item.json:209]:
```suggestion
   "fieldtype": "Column Break"
```

File: erpnext/stock/doctype/pick_list_item/pick_list_item.json [hunk 2]
```diff
@@ -200,10 +202,21 @@
...
+  },
+  {
+   "fieldname": "so_customer",
+   "fieldtype": "Data",
+   "label": "SO Customer",
+   "read_only": 1
```
Comment (sahil28297) [inline erpnext/stock/doctype/pick_list_item/pick_list_item.json:215]: where is `in_list_view`, `fetch_from`? Check the bloomstack_core file properly!!

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/stock/doctype/pick_list_item/pick_list_item.json (hunk 2 above)
Comment (mrunmay-epi) [reply erpnext/stock/doctype/pick_list_item/pick_list_item.json:215]: Hi Sahil,
I have made the suggested changes and raised the pr.

PR link: https://github.com/Bloomstack/erpnext/pull/1590

**Comment by sahil28297** [comment]
Duplicate

**Pull request closed without merging**